Automattic/jetpack
# Check if maxwidth exists before trying to call .length

Automattic/wp-calypso#58134

#### Changes proposed in this Pull Request:

This PR fixes an issue with the video block when handled by VideoPress causing errors on some existing block (see linked issue)

#### Does this pull request change what data or activity we track or use?
No

#### Testing instructions:
It seems a bit tricky to test, so, here is the procedure for wpcom

* On a sandboxed site with an existing post containing a VideoPress video
* By opening the post editor, the page shouldn't load
* Apply the wpcom patch
* ✅ Reloading the post should correctly load the Gutenberg editor
